src/core/variables.tf

```
@@ -1,15 +1,29 @@
-variable "location" {
-  type    = string
-  default = "westeurope"
-}
+# general
 
 variable "prefix" {
   type    = string
-  default = "pagopa"
+  default = "selc"
+  validation {
+    condition = (
+      length(var.prefix) > 6
+    )
+    error_message = "Max length is 6 chars."
+  }
 }
 
 variable "env_short" {
   type = string
+  validation {
+    condition = (
+      length(var.env_short) > 1
```

```suggestion
      length(var.env_short) <= 1
```
my error on template